SUMMARY.md

```diff
@@ -3,6 +3,7 @@
 * [Introduction](README.md)
   * [About this GitBook](introduction/prototyping_gitbook.md)
   * [Technical introduction](introduction/new_introduction.md)
+  * [Functional analysis](introduction/functional_analysis.md)
```

Comment:
are we sure it is the right place to put this file ? it might be located in the product backlog, what do you think ?

Reply:
For me it's better after technical introduction because it synthesize the information of the technical introduction in few schemes

Reply:
So, intro or backlog?

Reply:
let's go for intro ;) thanks @GG23800